api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -53,6 +53,17 @@ default String name() {
    */
   TableScan newScan();
 
+  /**
+   * Create a new {@link IncrementalTableScan scan} for this table.
+   * <p>
+   * Once a scan is created, it can be refined to project columns and filter data.
+   *
+   * @return an incremental scan for this table
+   */
+  default IncrementalTableScan newIncrementalScan() {
```

Comment:
Changing the `IncrementalTableScan` to `IncrementalScan` is matching the `newIncrementalScan` method name ?